torchvision/transforms/functional.py

```diff
@@ -1117,7 +1117,8 @@ def erase(img: Tensor, i: int, j: int, h: int, w: int, v: Tensor, inplace: bool
 def gaussian_blur(img: Tensor, kernel_size: List[int], sigma: Optional[List[float]] = None) -> Tensor:
     """Performs Gaussian blurring on the img by given kernel.
     The image can be a PIL Image or a Tensor, in which case it is expected
-    to have [..., H, W] shape, where ... means an arbitrary number of leading dimensions
+    to have [..., H, W] shape, where ... means an arbitrary number of leading dimensions.
+    If input is a PIL image, it will be first converted to Tensor for this operation, then converted back.
```

Comment:
```suggestion
```
The conversion from PIL => Tensor => PIL is an implementation detail which I don't think should be part of the documentation.

Reply:
Yes, you are right. I initially thought maybe some people who are concerned about speed might want to know about it.